packages/cargo_nextest/project.bri

```diff
@@ -32,18 +31,16 @@ export async function test(): Promise<std.Recipe<std.File>> {
 
   const result = (await script.read()).trim();
 
-  const version = result.split("\n").at(0);
-
   // Check that the result contains the expected version
   const expected = `cargo-nextest ${project.version}`;
-  std.assert(version === expected, `expected '${expected}', got '${version}'`);
+  std.assert(
+    result.startsWith(expected),
+    `expected '${expected}', got '${result}'`,
+  );
 
   return script;
 }
 
 export async function liveUpdate(): Promise<std.Recipe<std.Directory>> {
-  return std.liveUpdateFromGithubReleases({
```

Comment:
This is the advantage of checking from the crates.io, we don't need the `matchTag` here.